### PR TITLE
TOMEE-2890 Translate to Portuguese: examples/reload-persistence-unit

### PR DESCRIPTION
**Pull request by Daniel-Dos**
link to jira: https://issues.apache.org/jira/browse/TOMEE-2890

### CODE REVIEW

File: examples/reload-persistence-unit-properties/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+:jbake-type: page
+:jbake-status: status=published
+= Recarregar propriedades do persistence unit
+
+Este exemplo tem como objetivo simular uma campanha de benchmark no JPA.
+
+Primeiro, você executará seu aplicativo e perceberá que precisa de um L2
+cache para estar em conformidade com o SLA (Service Level Agreement).
+
+Então, você altera sua configuração persistence.xml e reinicia seu
```
Comment (hbelmiro) [inline examples/reload-persistence-unit-properties/README_pt.adoc:11]:
```suggestion
Então, você altera sua configuração persistence.xml e reinicia sua
```

File: examples/reload-persistence-unit-properties/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+O OpenEJB registra automaticamente um MBean por entitymanager (persistence unit).
+
+Ele permite que você altere as propriedades do persistence unit, mesmo que mais seja possível.
+
+== Testando
+
+O teste é simples: persistimos uma entidade, a consultamos três vezes
+sem cache, então ativamos o cache e realizamos a execução novamente
+consulta que um é suficiente para fazer o mesmo.
```
Comment (hbelmiro) [inline examples/reload-persistence-unit-properties/README_pt.adoc:27]: This sentence is confusing. Even in English it is not very clear.

Comment (Daniel-Dos) [reply examples/reload-persistence-unit-properties/README_pt.adoc:27]: I will re-read and see how I can improve.

thank you.

**Comment by cesarhernandezgt** [comment]
+1 from a technical point of view, the translation seems to require minor fixes but I don't know Portuguese.

**Pull request merged**